src/components/ui/multi-select.tsx

```
@@ -35,201 +39,208 @@ interface MultiSelectProps
-
-  const handleTouchMove = (e: React.TouchEvent) => {
-    e.stopPropagation();
-  };
-
   const triggerButton = (
     <Button
+      variant="outline"
       ref={ref}
-      {...props}
-      onClick={handleToggle}
+      role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)